server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/CreatedIds.scala

```diff
@@ -0,0 +1,33 @@
+/** **************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ***************************************************************/
+package org.apache.james.jmap.model
+
+import org.apache.james.jmap.model.CreatedIds.{ClientId, ServerId}
+import org.apache.james.jmap.model.Id.Id
+case class CreatedIds(value: Map[ClientId, ServerId])
+
+object CreatedIds {
+
+  final case class ClientId(value: Id)
+
+  final case class ServerId(value: Id)
```

Comment:
What's the added value of wrapping these case class in the CreatedIds object? Can't we move them outside?